src/packages/@app/pages/EnvironmentPage/EnvironmentPage.ViewModel.ts

```diff
@@ -327,9 +291,9 @@ export class EnvironmentViewModel {
       this.environmentRepository.updateEnvironment(env.id, {
         name: newEnvironmentName,
       });
-      this.environmentTabRepository.updateEnvironmentTab(env.id, {
-        name: newEnvironmentName,
-      });
+      let currenttab = await this.tabRepository.getTabById(env.id)
+      await this.tabRepository.updateTab(currenttab.tabId, { name: newEnvironmentName })
+
```

Comment:
add a check here if `currenttab` is not `undefined` or `null`

Reply:
Resolved